source/common/network/listener_impl.cc

```diff
@@ -58,39 +53,50 @@ ListenerImpl::ListenerImpl(Event::DispatcherImpl& dispatcher, Socket& socket, Li
   }
 
   if (bind_to_port) {
-    listener_.reset(
-        evconnlistener_new(&dispatcher.base(), listenCallback, this, 0, -1, socket.fd()));
+    const int ret = ::listen(socket.fd(), 128);
```

Comment:
I assume you looked at libevent code here and 128 is what it passed for the backlog? Maybe comment that?

Reply:
Use OsSysCalls, so this can be mocked/tested more easily?